src/status_im2/subs/chat/chats.cljs

```diff
@@ -213,6 +213,44 @@
                  :community-id
                  :emoji])))
 
+(re-frame/reg-sub
+ :chats/current-chat-message-list-view-context
+ :<- [:chats/current-chat-chat-view]
+ :<- [:communities/current-community]
+ :<- [:multiaccount/public-key]
+ (fn [[current-chat current-community current-public-key] [_ in-pinned-view?]]
+   (let [{:keys [group-chat chat-id public? admins space-keeper able-to-send-message?]}
+         current-chat
+
+         {:keys [can-delete-message-for-everyone? admin-settings]}
+         current-community
+
+         {:keys [pin-message-all-members-enabled?]} admin-settings
+         community? (some? current-community)
+         group-admin? (get admins current-public-key)
```

Comment:
In the `community?` binding we're using `some?` to get a boolean out of `current-community`. I think we should do the same to the bindings `group-admin?` and `community-admin?`, just to keep the returned map as lightweight as possible. But to be a bit more efficient, instead of `get`, we can directly use `contains?` like `(contains? admins current-public-key)`.

Reply:
fixed